packages/ui-extensions/docs/surfaces/checkout/reference/examples/ui-components/sheet-icon-button-preferences.example.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  extension,
+  Sheet,
+  Button,
+  Link,
+  TextBlock,
+  Icon,
+} from '@shopify/ui-extensions/checkout';
+
+export default extension(
+  'purchase.checkout.footer.render-after',
+  (root, {customerPrivacy}) => {
+    customerPrivacy.subscribe(
```

Comment:
Same comment regarding subscribing logic where this will happen every-time the customer privacy changes.